### PR TITLE
Refactor python converter functions to not call each other

### PR DESCRIPTION
**Pull request by mattsoulanille**
Make optimize_graph, extract_weights, and write_artifacts, which do separate things, no longer call each other.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7281)

### CODE REVIEW

File: tfjs-converter/python/tensorflowjs/converters/tf_saved_model_conversion_v2.py [hunk 1]
```diff
@@ -323,7 +278,7 @@ def extract_weights(graph_def,
                   initializer_signature_def=initializer_signature_def,
                   resource_ids_maps=resource_ids_maps,
                   metadata=metadata)
-
+ """
```
Comment (Linchenn) [inline tfjs-converter/python/tensorflowjs/converters/tf_saved_model_conversion_v2.py:281]: remove comments?

File: tfjs-converter/python/tensorflowjs/converters/tf_saved_model_conversion_v2.py [hunk 2]
```diff
@@ -222,18 +204,13 @@ def optimize_graph(graph, signature_def, output_graph,
...
-  extract_weights(
-      optimized_graph, output_graph, tf_version,
-      signature_def, quantization_dtype_map, weight_shard_size_bytes,
-      initializer_graph_def, initializer_signature_def,
-      resource_ids_maps=resource_ids_maps, metadata=metadata)
+  # extract_weights(
+  #     optimized_graph, output_graph, tf_version,
+  #     signature_def, quantization_dtype_map, weight_shard_size_bytes,
+  #     initializer_graph_def, initializer_signature_def,
+  #     resource_ids_maps=resource_ids_maps, metadata=metadata)
```
Comment (Linchenn) [inline tfjs-converter/python/tensorflowjs/converters/tf_saved_model_conversion_v2.py:213]: remove comments?

**Review by Linchenn — APPROVED** [review]
LGTM! Just some nits.

File: tfjs-converter/python/tensorflowjs/converters/tf_saved_model_conversion_v2.py (hunk 1 above)
Comment (mattsoulanille) [reply tfjs-converter/python/tensorflowjs/converters/tf_saved_model_conversion_v2.py]: Thanks! Sorry, I should have proof-read this better.

**Review by pyu10055 — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7281)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @Linchenn and @mattsoulanille)

**Pull request merged**